### ISSUE TITLE
[import] --users: cannot import admin group

### ISSUE THREAD

**Issue by nfx** [databrickslabs/migrate#7]
Import cannot re-create `admins` group:

```
Creating group: admins
Error: b'{"schemas":["urn:ietf:params:scim:api:messages:2.0:Error"],"detail":"Group with name admins already exists.","status":"409"}\n'
```

therefore it should not be exported.

**Comment by mrchristine** [comment]
No, we should be skipping already existing groups if they exist. I wouldn't skip the export. The export is needed for PVC and other envs. 
The reason it fails is because the Databricks CLI framework always raises an exception for anything not in the HTTP 2xx range, which is not ideal for this tool exactly for this scenario. We should remove that dependency or catch these exceptions for each call.
